src/handle.rs

```diff
@@ -102,6 +112,9 @@ impl Handle {
                     rv if rv >=0 => { nfq_handle_packet(self.ptr, buffer as *mut c_char, rv as i32); },
                     _ => { break; }
                 }
+                if self.processing {
```

Comment:
Can you wrap this into L112 (I'm not sure of this):
`rv if rv && self.processing`?